package.json

```diff
@@ -70,12 +70,14 @@
 		"@typescript-eslint/eslint-plugin": "^7.2.0",
 		"@typescript-eslint/parser": "^7.2.0",
 		"autoprefixer": "^10.4.19",
+		"cheerio": "^1.0.0-rc.12",
 		"dotenv": "^16.4.5",
 		"eslint": "^8.57.0",
 		"eslint-config-prettier": "^9.1.0",
 		"eslint-plugin-local-rules": "^2.0.1",
 		"eslint-plugin-svelte": "^2.35.1",
 		"jsdom": "^24.1.0",
+		"node-fetch": "^3.3.2",
```

Comment:
NodeJS now supports fetch out of the box.

Reply:
ah tks! i was in doubt